### ISSUE TITLE
Create appsettings.development.example.json in project detailing all settings used in application

### ISSUE THREAD

**Issue by chintans** [Promact/promact-oauth-server#38]
Create appsettings.development.example.json to provide a working example to the other developers how to create appsetting for development and production.

**Comment by chintans** [comment]
current appsettings does not follow best practices for configuration management is asp.net core. Please see below link to make it proper.

https://docs.asp.net/en/latest/fundamentals/configuration.html#using-options-and-configuration-objects

**Comment by ankit213** [comment]
Completed Create appsettings.development.example.Json file 
Remaining This Think "current appsettings does not follow best practices for configuration management is asp.net core. Please see below link to make it proper."

**Comment by juliepromact** [comment]
Analysed  the tutorial, which was mentioned in the above comment.
[https://docs.asp.net/en/latest/fundamentals/configuration.html#using-options-and-configuration-objects]

Found most of the code in StartUp.cs as per the tutorial. I commented out (but code not committed yet)
services.Configure<AppSettings>(options => Configuration.GetSection("AppSettings").Bind(options));
line of code, still able to obtain the environment variable value stored at launchSettings.json.
